plugins/reporters/spdx/src/main/kotlin/SpdxDocumentModelMapper.kt

```diff
@@ -61,14 +61,22 @@ internal object SpdxDocumentModelMapper {
         val nextFileIndex = AtomicInteger(1)
         val packages = mutableListOf<SpdxPackage>()
         val relationships = mutableListOf<SpdxRelationship>()
+        val files = mutableListOf<SpdxFile>()
 
         val projects = ortResult.getProjects(omitExcluded = true, includeSubProjects = false).sortedBy { it.id }
         val projectPackages = projects.map { project ->
+
+            val filesForProject = if (params.fileInformationEnabled) {
+                ortResult.getSpdxFiles(project.id, licenseInfoResolver, VCS, nextFileIndex)
```

Comment:
Somewhat related to my comment above:  You're using `VCS` as the source co0de origin here, so you should use the VCS location as the download location to make that match.

Reply:
I am not sure if you are implying that I need to do another change here, or that the change mentioned above makes sense due to using `VCS` here. I understand it as claiming that it makes sense based on using `VCS` information. So no change was performed.